os/services/rpl-border-router/embedded/Makefile.embedded

```diff
@@ -1,7 +1,13 @@
 PREFIX ?= fd00::1/64
 
+ifdef PORT
+	TUNSLIP6_FLAGS += -s $(PORT)
+endif
+
+TUNSLIP6_FLAGS += $(PREFIX)
```

Comment:
Thank you for the update; can you modify this as follows:
```
ifdef PORT
	TUNSLIP6_ARGS = -s $(PORT) $(PREFIX)
else
	TUNSLIP6_ARGS = $(PREFIX)
endif
```
Then, give $(TUNSLIP6_ARGS) to the line in `connect-router` rule instead of `$(TUNSLIP6_FLAGS)`. If you can test these changes with your device, it would be great.

After that, please squash commits into a single commit saying like "add PORT variable for 'make connect-router'"

Reply:
I have tested this with my cc1350 launchpad and it worked.

Reply:
> I have tested this with my cc1350 launchpad and it worked.
Great! Thank you.

Reply:
> Thank you for the update; can you modify this as follows:

Yes, it's what I was thinking too